real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/IMUTemperatureMessage.java

```diff
@@ -0,0 +1,61 @@
+package com.roboclub.robobuggy.messages;
```

Comment:
This file shouldn't be included, it's an artifact from the old code. Please remove it